tests/integration/components/suspense-component-test.js

```diff
@@ -245,4 +246,37 @@ module('Integration | Component | suspense-component', function (hooks) {
       'fastboot service was called correctly'
     );
   });
+
+  test('Component renders the loading state on the server correctly', async function (assert) {
+    const loadingSelector = '[data-test-async-loading]';
+
+    this.sandbox = sinon.createSandbox();
+    this.sandbox.stub(IS_BROWSER, 'default').value(false);
```

Comment:
I still need to upgrade sinon for directly using sinon.stub instead of createSandbox. Will do that in a separate PR